### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Select Subscription_Screen Reader: JAWS is giving wrong information regarding the checked and unchecked check-boxes

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3387]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!

**NOTE: Same is the case in Create Authentication Files dialog**

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

**Repro Steps:**
1) Start JAWS and open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Select Device login and tab till Sign in and press enter
8) Tab till 'Copy and Open' button and press enter
9) Paste the code in the browser and click on next
10) Sign-in with valid credentials
11) Go to IntelliJ IDEA application and tab till any checkbox in the table
12) Check if JAWS is reading the status of the check-box properly

**Actual Result:**
When the focus is on checked check-box, JAWS is reading as "not selected is true"

**Expected Result:**
JAWS should give the correct information regarding the checked/unchecked check-box

**User Impact:**
Screen Reader users will be provided with wrong information

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=35e0fa4e-05f4-483e-98b5-2ddae34a2404)

[Bug_3387_IntelliJ_Select Subscription_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3466947/Bug_3387_IntelliJ_Select.Subscription_JAWS.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been fixed in #3515 , here is the [build](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_PR/3778/Azure/processDownloadRequest/artifacts/WindowsAzurePlugin4EJ.PR.3778.09-10-2019.zip), please help us verify it , thanks!

**Comment by soumyatippapuram** [comment]
Regressed this issue in the build provided above. This issue is fixed. Hence, closing this. Added attachment for reference. 
[Bug_3387_Jaws_Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3595536/Bug_3387_Jaws_Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749777
